ROADMAP.adoc

```diff
@@ -0,0 +1,45 @@
+= jenkins.io roadmap
+
+This document is the home of the roadmap for moving from the "legacy" site on link:https://jenkins-ci.org[jenkins-ci.org]
+to the new and improved, super amazing, link:https://jenkins.io[jenkins.io]
+
+
+== Static Site switchover
+
+1. properly import non-story (i.e. standalone) pages from Drupal (link:https://issues.jenkins-ci.org/browse/WEBSITE-34[WEBSITE-34])
+1. import legacy site (*on-going until full cut-off*)
+1. determine where we can generate the site (link:https://issues.jenkins-ci.org/browse/WEBSITE-35[WEBSITE-35])
+1. determine how we're going to host (link:https://issues.jenkins-ci.org/browse/WEBSITE-38[WEBSITE-38])
+1. deploy the new statically generated site to link:http://beta.jenkins-ci.org[beta.jenkins-ci.org]
+1. cut over from Drupal to the statically generated site
```

Comment:
I'd probably rephrase it as `cut over jenkins-ci.org from Drupal to the statically generated site` and create a ticket for that so that people can post thoughts if any.